src/service_name.json

```diff
@@ -0,0 +1,514 @@
+  },
+  {
+    "Command": "az sentinel",
+    "Description": "Manage security and events",
+    "AzureServiceName": "Azure Sentinel",
+    "ConversationalServiceName": "Sentinel",
+    "ExtensionName": "sentinel",
+    "URL": "https://docs.microsoft.com/azure/sentinel/overview"
+  },
+  {
+    "Command": "az spatial-anchors-account",
+    "Description": "Manage Spatial Anchors Accounts",
+    "AzureServiceName": "Azure Spatial Anchors",
+    "ConversationalServiceName": "Spatial Anchors",
+    "ExtensionName": "mixed-reality",
+    "URL": "https://docs.microsoft.com/windows/mixed-reality/mixed-reality"
+  },
+  {
+    "Command": "az spring-cloud",
+    "Description": "Microsoft Azure Command-Line Tools spring-cloud Extension",
+    "AzureServiceName": "Azure Spring Cloud",
+    "ConversationalServiceName": "Spring Cloud",
+    "ExtensionName": "spring-cloud",
+    "URL": "https://docs.microsoft.com/azure/spring-cloud/"
+  },
+  {
+    "Command": "az ssh",
+    "Description": "Use secure shell (SSH)with Azure VMs.",
+    "AzureServiceName": "Azure Virtual Machines",
+    "ConversationalServiceName": "Virtual Machines",
+    "ExtensionName": "ssh",
+    "URL": "https://docs.microsoft.com/azure/virtual-machines/linux/mac-create-ssh-keys/"
+  },
+  {
+    "Command": "az stack-hci",
+    "Description": "Commands to manage stack-hci clusters.",
+    "AzureServiceName": "Azure Stack HCI",
+    "ConversationalServiceName": "Stack HCI",
+    "ExtensionName": "stack-hci",
+    "URL": "https://docs.microsoft.com/azure-stack/hci/"
+  },
+  {
+    "Command": "az storagesync",
+    "Description": "Microsoft Azure Command-Line Tools MicrosoftStorageSync Extension",
+    "AzureServiceName": "Azure Storage",
+    "ConversationalServiceName": "Storage",
+    "ExtensionName": "storagesync",
+    "URL": "https://docs.microsoft.com/azure/storage/files/storage-sync-files-planning"
+  },
+  {
+    "Command": "az stream-analytics",
+    "Description": "Microsoft Azure Command-Line Tools stream-analytics Extension",
+    "AzureServiceName": "Azure Stream Analytics",
+    "ConversationalServiceName": "Stream Analytics",
+    "ExtensionName": "stream-analytics",
+    "URL": "https://docs.microsoft.com/azure/stream-analytics/"
+  },
+  {
+    "Command": "az support",
+    "Description": "Microsoft Azure Command-Line Tools Support Extension",
+    "AzureServiceName": "Azure Support",
+    "ConversationalServiceName": "Support",
+    "ExtensionName": "support",
+    "URL": "https://azure.microsoft.com/support/options/"
+  },
+  {
+    "Command": "az timeseriesinsights",
+    "Description": "Microsoft Azure Command-Line Tools TimeSeriesInsightsClient Extension",
+    "AzureServiceName": "Azure Time Series Insights",
+    "ConversationalServiceName": "Time Series Insights",
+    "ExtensionName": "timeseriesinsights",
+    "URL": "https://docs.microsoft.com/azure/time-series-insights/"
+  },
```

Comment:
This command group is deprecated. Now its name is "az tsi" which already included below.
```suggestion
```